webpack/src/Components/EmptyState/EmptyStateReducer.js

```diff
@@ -0,0 +1,16 @@
+import Immutable from 'seamless-immutable';
+import { ADD_CONTENT } from './Constants';
+
+const initialState = Immutable({});
+
+export default (state = initialState, action) => {
+  const { payload } = action;
+
+  switch (action.type) {
+    case ADD_CONTENT:
+      return state.set('header', payload);
+
```

Comment:
nit: extra line

Reply:
our linting rules allowed this kind of spacing